intelmq/bots/experts/filter/expert.py

```diff
@@ -3,20 +3,75 @@
 import sys
 
 from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import DateTime
+from datetime import datetime, timedelta
+from dateutil import parser
+import re
```

Comment:
Is a built-in packages, should go in line before sys-import. You may use `isort -rc filename` (python package from pip)